src/app/components/shared/detail-view/render-field/render-field.component.ts

```diff
@@ -17,18 +17,17 @@ import { DateTime, Duration } from "luxon";
 import { Observable } from "rxjs";
 import { takeUntil } from "rxjs/operators";
 
-// TODO Simplify this component by splitting into sub components
 @Component({
   selector: "baw-render-field",
   template: `
     <!-- Display plain text -->
     <dl *ngIf="styling === fieldStyling.plain">
-      <p id="plain" class="m-0">{{ display }}</p>
+      <p id="plain" class="m-0" [innerText]="display"></p>
```

Comment:
is there a reason for using innerText here?

Reply:
Using `{{ display }}` adds random whitespace to the contents which is annoying for tests